sources/packages/backend/apps/workers/src/controllers/assessment/assessment.dto.ts

```diff
@@ -82,6 +82,14 @@ export interface ApplicationAssessmentJobOutDTO {
    * Application associated with this application.
    */
   applicationId: number;
+  /**
+   * Current status of the application.
+   */
+  applicationStatus: string;
+  /**
+   * If any NOA has been approved for the application.
+   */
+  noaApproval: boolean;
```

Comment:
Can we rename it to `hasNOAApproval`?
 The idea is to use `is` or `has` for boolean variables.

Reply:
Updated